silverstripe/comments/SS-2018-015-1.yaml

```diff
@@ -0,0 +1,14 @@
+title:     'SS-2018-015: Vulnerable dependency'
+link:      https://www.silverstripe.org/download/security-releases/ss-2018-015/
+cve:       ~
+branches:
+    1.x:
+        time:     null
+        versions: ['>=1.3.0', '<1.99.99']
```

Comment:
Would the following be more accurate? I can't find details from FriendsOfPHP/security-advisories on the best approach here.

```suggestion
        versions: ['>=1.3.0', '<2']
```

Reply:
Needs to be semver so can't use `<2`, though yeah the .99.99 is a bit weird.  I'll change to `<1.9.9` which is functionally the same